core/templates/dev/head/base.html

```diff
@@ -50,7 +51,7 @@
     <meta property="og:image" content="{{DOMAIN_URL}}/images/logo/288x288_logo_mint.png">
 
     <link rel="apple-touch-icon" href="/images/logo/favicon.png">
-    <title itemprop="name">{% block subtitle %}{% endblock %} - {% block maintitle %}Oppia{% endblock %}</title>
+    <title itemprop="name" ng-bind="pageTitle">{% block subtitle %}{% endblock %} - {% block maintitle %}Oppia{% endblock %}</title>
```

Comment:
Can you add a comment above explaining why this is needed? (Then one day we might try and get rid of the Jinja title/subtitle specification, but at least this makes it clear there's an overwrite -- if I'm understanding correctly.)